cypress/support/index.js

```diff
@@ -14,7 +14,7 @@
 // ***********************************************************
 
 // Import commands.js using ES2015 syntax:
-import './commands'
+import './commands';
 
 // Alternatively you can use CommonJS syntax:
```

Comment:
I think commented lines do not look good in a codebase.

Reply:
> I think commented lines do not look good in a codebase.

These are not part of my feature,it's cypress configuration

Reply:
They are no commented lines of code,they are comments instead